ContextualPopupDecorator/ContextualPopup.js

```diff
@@ -196,8 +205,10 @@ const ContextualPopupBase = kind({
 			{reserveClose: showCloseButton},
 			offset
 		),
-		closeButton: ({css, showCloseButton, onCloseButtonClick}) => {
+		closeButton: ({css, showCloseButton, skin, onCloseButtonClick}) => {
 			if (showCloseButton) {
+				const closeButtonSize = skin === 'silicon' ? 'smallest' : 'small';
```

Comment:
Don't you think it's too small? Please confirm with GUI person the size is correct. I think it's too small to touch..
![image](https://user-images.githubusercontent.com/5037429/116196062-f8a88680-a76d-11eb-9da3-613621fa5067.png)

Reply:
The GUI was updated and now it's "small" size for every skin. The font-size was also enlarged.

Reply:
👏 The change looks great! Thanks!